tests/functional/Parser/Fixtures/Assignments/include_dir_condition.typoscript

```diff
@@ -0,0 +1,3 @@
+<INCLUDE_TYPOSCRIPT: source="DIR:foo/" condition="Your\Custom\ClassOne">
```

Comment:
Notice that class-based custom conditions need to be written with backslashes escaped by backslashes, in this case `Your\\Custom\\ClassOne`. I currently try to find the source again but I also noticed that my condition simply didn't work with a single backslash.